packages/astro/src/core/ssr/result.ts

```diff
@@ -34,6 +38,26 @@ export function createResult(args: CreateResultArgs): SSRResult {
 					params,
 					url,
 				},
+				resolve(path: string) {
+					if(astroConfig.buildOptions.experimentalStaticBuild) {
+						let extra = `This can be replaced with a dynamic import like so: await import("${path}")`;
```

Comment:
Does this need `?url`?

Reply:
Not for images at least. I'm not sure when it's needed or not, actually. I guess we could look for common image extensions and a more specific warning like with CSS, and then fallback to using `?url`.

Reply:
Up to you, not blocking!

Reply:
Ok, I think we'll leave it like this for now as I'm not certain what situations require using `?url`. Can add that later.